src/components/index.tests.tsx

```diff
@@ -383,8 +383,14 @@ class SelectTest extends React.Component<any, any> {
           options={options}
           isAsync={true}
           isCreatable={true}
+          data-tid="foo"
+        />
+        <Select
```

Comment:
I should mention to you (and I've said this in various forums so you may or may not have heard it) but Select has an issue in that it doesn't work with the form engine validation properly. In practice, this hasn't been an issue for the most part because we don't usually need to validate what's selected per se, but, if you do, please be aware this is a "known bug" if not the most serious one in EDS cc / @anbright

Reply:
Alright, thanks for the heads up!